packages/prosemirror-lwdita/src/plugin.ts

```diff
@@ -323,3 +298,29 @@ export function menu(schema: Schema, { start, before, after, end}: Additions = {
     ...end,
   ] });
 }
+
+
+export const doubleClickImagePlugin = new Plugin({
+  props: {
+    handleDOMEvents: {
+      dblclick: (view, event) => {
+        const { schema, doc } = view.state;
+        const pos = view.posAtCoords({ left: event.clientX, top: event.clientY });
+
+        if (pos) {
```

Comment:
@marmoure I have found a special case where if the cursor is anywhere in the `topic/title` element and you insert an image, a whole new topic node (aka html article) is created which destroys the original document structure because it effectively creates 2 new documents.

Reply:
This show the inherent issues in the schema, It will be fixed once the schema is enforced more strictly.

Reply:
Ok, so we have to keep this in mind and mark treat is as a won't fix for now.